src/Weft/Generics/Resolve.hs

```diff
@@ -86,8 +87,8 @@ instance ResolveField (IO scalar)
 instance {-# OVERLAPPING #-}
          ( KnownSymbol n
          , ResolveField rv (Args args, ru) rp
-         ) => ResolveField (Arg n (Maybe t) -> rv)
-                           (Args ('(n, Maybe t) ': args), ru)
+         ) => ResolveField (Arg n (M.Map Text t) -> rv)
```

Comment:
This thing shouldn't change. BUT ALSO we don't need this special case now after #35